code/datums/mutations/body.dm

```diff
@@ -230,6 +231,108 @@
 		return
 	owner.physiology.burn_mod *= 2
 
+/datum/mutation/human/badblink
+	name = "Spatial Instability"
+	desc = "Subject has a very weak link to spatial reality, and may be displaced. Often causes extreme nausea."
```

Comment:
Just FYI the mutation desc usually has the mutation itself as the subject rather than the person which manifests such a mutation. It's really triggering the perfectionist within.